test/components/auto-complete.spec.tsx

```diff
@@ -110,33 +109,6 @@ describe('<AutoComplete />', () => {
         });
     });
 
-    it('places the caret inside the input and centers it', async () => {
-        const ThemedAutoComplete = WithTheme(<AutoComplete />);
-        const {select, waitForDom} = clientRenderer.render(<ThemedAutoComplete/>);
-
-        await waitForDom(() => {
-            const autocomplete = select(autoComp)!;
-            const input = select(autoComp, autoCompInput)!;
-            const caret = select(autoComp, autoComp + '_CARET')!;
-
-            expect(caret).to.be.insideOf(autocomplete);
-            expect([input, caret]).to.be.verticallyAligned('center');
-        });
-    });
-
-    it('calls the onOpenStateChange event when clicking on the caret', async () => {
```

Comment:
This one should pass without a theme.

Reply:
nope. caret by default has no size, this is exact test that was considered as theme-test when I escalated this problem to @idoros 

If you feel, lets make a call together again

Reply:
Yes, it has no size, but I don't see it as a problem. We can still simulate the click. We should simply change `.be.present()` to `.be.not.null`.

Reply:
I agree.this is a functional test that has nothig to do with theme